auv_control.orogen

```diff
@@ -67,7 +67,7 @@ task_context "Base" do
     dynamic_input_port(/cmd_\w+/, "base::LinearAngular6DCommand")    
 
     runtime_states :CONTROLLING, :CONTROLLING_UNSAFE
-    error_states :INPUT_MISSING, :INPUT_COLLIDING, :INPUT_UNEXPECTED, :TIMEOUT, :WAIT_FOR_CONNECTED_INPUT_PORT, :WAIT_FOR_INPUT 
+    error_states :INPUT_MISSING, :INPUT_COLLIDING, :INPUT_UNEXPECTED, :TIMEOUT, :WAIT_FOR_CONNECTED_INPUT_PORT, :WAIT_FOR_INPUT, :ERROR_CALCOUTPUT
```

Comment:
Don't create generic error states like `ERROR_CALCOUTPUT`. If you need one, you can simply transition to the generic RTT error state.